### PR TITLE
Fix grammar in troubleshoot.rst

### PR DESCRIPTION
**Pull request by armgabrielyan**
Fix grammar in documentation on [Troubleshooting and tips](https://numba.readthedocs.io/en/stable/user/troubleshoot.html).

### CODE REVIEW

File: docs/source/user/troubleshoot.rst [hunk 1]
```diff
@@ -874,7 +874,7 @@ breakpoint was hit, and after a ``continue`` was issued, it broke again at line
 Debugging in parallel regions
 -----------------------------
 
-The follow example is quite involved, it executes with ``gdb`` instrumentation
+The following example is quite involved, it executes with ``gdb`` instrumentation
```
Comment (stuartarchibald) [inline docs/source/user/troubleshoot.rst:877]: Please could this be line wrapped to 80 characters? Many thanks.

**Review by stuartarchibald — COMMENTED** [review]
Thanks for fixing these typos, one thing to resolve RE linewrap else look good.

File: docs/source/user/troubleshoot.rst (hunk 1 above)
Comment (armgabrielyan) [reply docs/source/user/troubleshoot.rst]: Thanks for the comment! I have made the appropriate changes. Could you please check it out?

**Review by stuartarchibald — APPROVED** [review]
Thanks for the patch and fixes!

**Comment by armgabrielyan** [comment]
Thank you!

**Pull request merged**